src/cloud.c

```diff
@@ -349,7 +349,7 @@ int cloud_unregister_device(const char *id)
 int cloud_auth_device(const char *id, const char *token)
 {
 	amqp_bytes_t queue_cloud;
-	json_object *jobj_device;
```

Comment:
About commit message:
This patch changes **json_object** name on cloud_auth_device **to** a more
suitable one. This **json_object** **represents** **an** auth message.

Reply:
Ok